Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -286,12 +302,23 @@ public double coneStackWidthMotorPower(double coneStackWidth) {
     }
 
     /**
-     * calculates the motor power for the robot drivetrain based on the pixel offset from the cone stack
+     * calculates the turning power for the robot drivetrain based on the pixel offset from the cone stack and the width of the cone stack
      * @param pixelOffset how far cone stack is horizontally from center of camera field of view in pixels
-     * @return motor power for robot drivetrain
+     * @param width how wide the cone stack is in the camera frame in pixels
+     * @return turning power for robot drivetrain
+     */
+    public double coneStackPixelsAndWidthToTurningPower(double pixelOffset, double width) {
+        return (Constants.CONE_STACK_CENTERING_PROPORTIONAL_KP * pixelOffset)/((1.0/Constants.TURNING_AUTHORITY_CONSTANT)*width);
```

Comment:
Be consistent in your spacing and make sure you have spaces around operators
```suggestion
        return (Constants.CONE_STACK_CENTERING_PROPORTIONAL_KP * pixelOffset) / ((1.0 / Constants.TURNING_AUTHORITY_CONSTANT) * width);
```